lib/ansible.js

```diff
@@ -24,10 +24,18 @@ AbstractAnsibleCommand.prototype.exec = function(options) {
 
   var deferred = Q.defer();
   var processOptions = {};
-  if (options && options.cwd) {
+  if (!options) options = {};
+
+  if (options.cwd) {
     processOptions.cwd = options.cwd;
   }
 
+  // Make Ansible output unbuffered so the stdout and stderr `data` events 
+  // pickup output up more regularly
+  var unbuffered = (options.buffered) ? '' : 1;
+  processOptions.env = process.env;
```

Comment:
I was thinking about that when I hit the issue, It was really only `PATH` that was needed to launch `ansible-playbook` but I went with the whole environment as I'm sure there could be other vars that ansible might rely on. 

For example when initially when testing this, I expected that `PYTHONUNBUFFERED` in the shell launching `node` would be passed through. You could build up that list of vars to be passed through but things will break in the mean time.